tests/Riverline/MultiPartParser/StreamedPartTest.php

```diff
@@ -58,20 +58,36 @@ public function testNoMultiPart()
     }
 
     /**
-     * Test a simple multipart document
+     * Test that is not possible to get a body of a multi part document
      */
-    public function testSimpleMultiPart()
```

Comment:
This test has been split in two, as it was hiding other errors when parsing parts.